common/contracts/rust/Cargo.toml

```diff
@@ -5,10 +5,10 @@ authors = ["Near Inc <hello@near.org>"]
 edition = "2018"
 
 [lib]
-crate-type = ["cdylib", "rlib"]
+crate-type = ["cdylib"]
 
 [dependencies]
-near-sdk = "3.1.0"
+near-sdk = "4.0.0-pre.1"
```

Comment:
Should we wait for the release of 4.0.0?

Reply:
@mikedotexe maybe you have an opinion here. This release is intended to be stable, without any core fundamental changes. Figured it would be a good way to get users on this version before we release the stable version, and it allows users to have the most up-to-date APIs and without extra bloat.

Reply:
Okay, thanks for clarifying. Yes, I'm down to use prereleases like this. We haven't done this before, but I'm on board. Previously we've been using prereleases as a kind of hack to (if I recall) make sure CI passes across the board with different projects using it.

Do you have concerns, @volovyk-s ?

Reply:
After the last clarification from @austinabell, I have no concerns anymore.

Reply:
But, just for posterity, I do believe we should avoid updating this to pre-releases in the future, but in this case, there are a few rough edges on the API we want to move away from ASAP before users get used to it.